packages/kad-dht/src/peer-list/peer-distance-list.ts

```diff
@@ -1,9 +1,11 @@
-import { convertPeerId, getDistance } from '../utils.js'
-import type { PeerId } from '@libp2p/interface'
+import { xor as uint8ArrayXor } from 'uint8arrays/xor'
+import { xorCompare as uint8ArrayXorCompare } from 'uint8arrays/xor-compare'
+import { convertPeerId } from '../utils.js'
+import type { PeerId, PeerInfo } from '@libp2p/interface'
 
 interface PeerDistance {
-  peerId: PeerId
-  distance: bigint
+  peer: PeerInfo
+  distance: Uint8Array
```

Comment:
If this ends up getting [converted to a `bigint`](https://github.com/libp2p/js-libp2p/blob/main/packages/kad-dht/src/peer-list/peer-distance-list.ts#L99) what's the rationale behind changing these to `Uint8Array` ?

Reply:
You're linking to main there - it no longer gets converted to a bigint, instead the `Uint8Array`s are the xor distances that get compared using `xorCompare`.